node/src/components/small_network.rs

```diff
@@ -1076,6 +1103,12 @@ where
 
                 effects
             }
+            Event::ChainSynchronizerAnnouncement(ChainSynchronizerAnnouncement::SyncFinished) => {
+                info!("node has finished syncing");
```

Comment:
This is not a message we should put out - it belongs in fast sync.

I'd support a (potentially debug) style message that states that the networking component was _informed_ about sync completion though.